frontends/web/src/api/keystores.ts

```diff
@@ -31,3 +31,34 @@ export const subscribeKeystores = (
 export const getKeystores = (): Promise<TKeystores> => {
   return apiGet('keystores');
 };
+
+export type TSyncConnectKeystore = null | {
+  typ: 'connect';
```

Comment:
isn't it `type` instead of `typ` ?

Reply:
Nope, I think this is done on purpose to avoid collisions with protected keywords. It is defined like this in the backend https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/backend/accounts.go#L579-L584